server/system/environment.js

```diff
@@ -0,0 +1,13 @@
+const R = require('ramda')
+
+const keys = {
+  tempFolder: 'TEMP_FOLDER'
+}
+
+const defaultTempFolder = '/tmp/arena_upload'
+
+const tempFolder = R.propOr(defaultTempFolder, keys.tempFolder)(process.env)
+
+module.exports = {
+  tempFolder
+}
```

Comment:
what about adding to common/processUtils and ENV object that contains all the env variables? this mean all references to process.env should be replace by ```ProcessUtils.ENV.[var]```?

Reply:
honestly I don't like the "utils" prefix, and I would prefer to have a getter for every environment variable

Reply:
why a getter ?